code/modules/mining/lavaland/loot/colossus_loot.dm

```diff
@@ -291,9 +291,9 @@
 			L.heal_overall_damage(heal_power, heal_power)
 			new /obj/effect/temp_visual/heal(get_turf(target), "#80F5FF")
 
-/mob/living/simple_animal/hostile/lightgeist/ghostize()
-	if(..())
-		death()
+/mob/living/simple_animal/hostile/lightgeist/ghost()
+	qdel(src)
+	return ..()
```

Comment:
is there a reason why `qdel()` is called here before the parent? In the parent it prompts the player beforehand where in this case it will immediately kill the player

Reply:
I don't even need to call parent here, qdeleting them works fine. I assume the purpose of this interaction is to let them suicide even though they're simple mobs so it's handled the same way.